docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+## Configuring RHACM Agent Registration
+
+Auto-registration of devices with MicroShift into ACM relies on a feature called agent registration, allowing clusters to be imported via REST API calls using a CA bundle and token. Follow these [instructions](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#importing-managed-agent) to configure the necessary RBAC policies for your own user. Skip step 7 of the instructions in section 1.6.5.2.3, as this step will be automatically handled by the fleet devices.
+
+As part of this setup, you will need to collect the following information:
+
+- Agent registration URL
+- Token
+
+## Creating Repositories in Flight Control
```

Comment:
How to use the HTTP Config Provider is already explained in section
https://github.com/flightctl/flightctl/blob/main/docs/user/managing-devices.md#getting-configuration-from-an-http-server. I see it lacks the generic description of how to set up a corresponding Repo.

So here we shouldn't repeat how to set up repo or configure the provider, but just tell the user that these need to be created with X and Y as parameters.

Reply:
Ok, makes sense.